_config_base.yml

```diff
@@ -79,32 +79,31 @@ release_info:
     version: v20.2.19
   v21.1:
     build_time: 2022-05-09 00:00:00 (go1.15)
+    crdb_branch_name: release-21.1
     docker_image: cockroachdb/cockroach
     name: v21.1.19
     start_time: 2022-05-10 14:15:58.748869 +0000 UTC
     version: v21.1.19
-    crdb_branch_name: release-21.1
   v21.2:
     build_time: 2022-08-29 00:00:00 (go1.16)
+    crdb_branch_name: release-21.2
     docker_image: cockroachdb/cockroach
     name: v21.2.15
     start_time: 2022-08-26 14:48:15.734048 +0000 UTC
     version: v21.2.15
-    crdb_branch_name: release-21.2
   v22.1:
     build_time: 2022-08-23 00:00:00 (go1.17)
+    crdb_branch_name: release-22.1
     docker_image: cockroachdb/cockroach
     name: v22.1.6
     start_time: 2022-08-22 14:17:14.803026 +0000 UTC
     version: v22.1.6
-    crdb_branch_name: release-22.1
   v22.2:
-    build_time: 2022-08-30 00:00:00 (go1.17)
+    build_time: 2022-09-06 00:00:00 (go1.17)
     docker_image: cockroachdb/cockroach-unstable
-    name: v22.2.0-alpha.1
-    start_time: 2022-08-26 10:55:11.692728 +0000 UTC
-    version: v22.2.0-alpha.1
-    crdb_branch_name: master
```

Comment:
just verifying that we indeed want to remove the `crdb_branch_name` field from `v22.2` (guessing because it's easily computed as `master` anyways)